velox/functions/sparksql/String.h

```diff
@@ -759,6 +759,28 @@ struct LeftFunction {
   }
 };
 
+template <typename T>
+struct BitLengthVarcharFunction {
```

Comment:
Can we add this function following alphabetical order?

Reply:
Done.